pkg/volume/flexvolume/driver-call.go

```diff
@@ -124,7 +128,35 @@ func (dc *DriverCall) Run() (*DriverStatus, error) {
 		defer timer.Stop()
 	}
 
-	output, execErr := cmd.CombinedOutput()
```

Comment:
Would it be possible to encapsulate your new stuff in a version of CombinedOutput that returns two byte sequences? This looks like something that would be generally useful.